patches/0010-add-appinsights-telemetry.patch

```diff
@@ -5,14 +5,13 @@ Subject: [PATCH] Add appinsights telemetry
 
 ---
  README.md                                     | 15 ++++++
- .../internal/telemetrystats/telemetrystats.go | 13 +++++
```

Comment:
This one tool me a minute to work out but @qmuntal upstreamed this in https://github.com/golang/go/commit/19a086f716869bc79df24e1f30d74334261396db